http-server-jetty/src/main/java/io/micronaut/servlet/jetty/JettyConfiguration.java

```diff
@@ -168,44 +165,60 @@ public static class JettySslConfiguration extends SecureRequestCustomizer {
      */
     @ConfigurationProperties(JettyRequestLog.ACCESS_LOG)
     @Requires(property = JettyRequestLog.ENABLED_PROPERTY, value = StringUtils.TRUE)
-    public static class JettyRequestLog implements Toggleable {
+    public static final class JettyRequestLog implements Toggleable {
         public static final String ACCESS_LOG = "access-log";
         public static final String ENABLED_PROPERTY = HttpServerConfiguration.PREFIX + ".jetty." + ACCESS_LOG + ".enabled";
-        @ConfigurationBuilder(prefixes = "set", excludes = "eventListeners")
-        RequestLogWriter requestLogWriter = new RequestLogWriter();
 
         private boolean enabled = true;
-        private String pattern = CustomRequestLog.EXTENDED_NCSA_FORMAT;
```

Comment:
this is a breaking change. I think we need to keep this and instead alter have 2 configuration options one for logback and one for the default when logback-access is not on the classpath.